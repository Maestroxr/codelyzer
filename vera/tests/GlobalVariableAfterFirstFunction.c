
int main()
{
	/*OK*/
	int returnValue =0;
	/*PROBLEM*/
	return returnValue;
	
}
/*PROBLEM*/
int x = 0;
/*********/
