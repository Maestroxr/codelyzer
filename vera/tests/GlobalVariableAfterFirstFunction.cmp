C:\Work\AutoAssessment\codelyzer\vera\tests\GlobalVariableAfterFirstFunction.c:11: Global variable 'problem' appears after the first function main.