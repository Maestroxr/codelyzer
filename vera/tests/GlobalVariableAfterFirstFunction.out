C:\Work\AutoAssessment\codelyzer\vera\tests\GlobalVariableAfterFirstFunction.c:11: 'Global variable 'x' appears after the first function main.
